circuit/types/fullexit_nft.go

```diff
@@ -69,17 +69,20 @@ func SetFullExitNftTxWitness(tx *FullExitNftTx) (witness FullExitNftTxConstraint
 func VerifyFullExitNftTx(
 	api API, flag Variable,
 	tx FullExitNftTxConstraints,
-	accountsBefore [NbAccountsPerTx]AccountConstraints, nftBefore NftConstraints,
+	accountsBefore [NbAccountsPerTx]AccountConstraints,
+	nftBefore NftConstraints,
 ) (pubData [PubDataBitsSizePerTx]Variable) {
+	fromAccount := 0
+	creatorAccount := 1
+
 	pubData = CollectPubDataFromFullExitNft(api, tx)
 	// verify params
-	IsVariableEqual(api, flag, tx.AccountNameHash, accountsBefore[0].AccountNameHash)
-	IsVariableEqual(api, flag, tx.AccountIndex, accountsBefore[0].AccountIndex)
+	IsVariableEqual(api, flag, tx.AccountNameHash, accountsBefore[fromAccount].AccountNameHash)
+	IsVariableEqual(api, flag, tx.AccountIndex, accountsBefore[fromAccount].AccountIndex)
 	IsVariableEqual(api, flag, tx.NftIndex, nftBefore.NftIndex)
-	isCheck := api.IsZero(api.IsZero(tx.CreatorAccountNameHash))
-	isCheck = api.And(flag, isCheck)
-	IsVariableEqual(api, isCheck, tx.CreatorAccountIndex, nftBefore.CreatorAccountIndex)
-	IsVariableEqual(api, isCheck, tx.CreatorTreasuryRate, nftBefore.CreatorTreasuryRate)
+	IsVariableEqual(api, flag, tx.CreatorAccountIndex, accountsBefore[creatorAccount].AccountIndex)
+	IsVariableEqual(api, flag, tx.CreatorAccountIndex, nftBefore.CreatorAccountIndex)
+	IsVariableEqual(api, flag, tx.CreatorTreasuryRate, nftBefore.CreatorTreasuryRate)
```

Comment:
It seems you remove this constraint wrongly:`IsVariableEqual(api,flag, tx.CreatorAccountNameHash, accountsBefore[creatorAccount].AccountNameHash)`